app/models/user.rb

```diff
@@ -13,19 +13,15 @@ def self.member(id)
   aasm whiny_transitions: false do
     state :new, initial: true
     state :transitioned
+    # TODO:  Remove all these after migrating users all to "transitioned",
```

Comment:
I've started a ticket for this https://trello.com/c/6wQFnuhP/1443-migrate-users-to-transitioned if you want to remove the TODO. No problem if you want to leave it there for context though.